src/components/NetworkQualityLevel/NetworkQualityLevel.tsx

```diff
@@ -1,42 +1,53 @@
 import React from 'react';
-import { makeStyles } from '@material-ui/core/styles';
+import clsx from 'clsx';
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
 import { Participant } from 'twilio-video';
+import { useAppState } from '../../state';
 import useParticipantNetworkQualityLevel from '../../hooks/useParticipantNetworkQualityLevel/useParticipantNetworkQualityLevel';
 
-const useStyles = makeStyles({
-  outerContainer: {
-    width: '2em',
-    height: '2em',
-    padding: '0.9em',
-    display: 'flex',
-    alignItems: 'center',
-    justifyContent: 'center',
-    background: 'rgba(0, 0, 0, 0.5)',
-  },
-  innerContainer: {
-    display: 'flex',
-    alignItems: 'flex-end',
-    '& div': {
-      width: '2px',
-      marginRight: '1px',
-      '&:not(:last-child)': {
-        borderRight: 'none',
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    outerContainer: {
+      width: '2em',
+      height: '2em',
+      padding: '0.9em',
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'center',
+      background: 'rgba(0, 0, 0, 0.5)',
+    },
+    innerContainer: {
+      display: 'flex',
+      alignItems: 'flex-end',
+      '& div': {
+        width: '2px',
+        marginRight: '1px',
+        '&:not(:last-child)': {
+          borderRight: 'none',
+        },
+      },
+    },
+    mobileGridView: {
+      [theme.breakpoints.down('sm')]: {
+        width: '1.5em',
+        height: '1.5em',
       },
     },
-  },
-});
+  })
```

Comment:
So I came across this fix accidentally, but it turns out that this change here isn't needed. When `box-sizing: border-box` is added in MobileGridView, it gets applied to this component as well, which completely changes the size of the element. I think it's because the default style for `box-sizing` on a div is `inherit`.